Modules/Loadable/Markups/MRML/vtkMRMLMarkupsDisplayNode.h

```diff
@@ -443,6 +443,17 @@ class  VTK_SLICER_MARKUPS_MODULE_MRML_EXPORT vtkMRMLMarkupsDisplayNode : public
   vtkSetMacro(InteractionHandleScale, double);
   vtkGetMacro(InteractionHandleScale, double);
 
+  /// Get/Set the visibility of the individual handle axes
```

Comment:
To associate this documentation block with all the Get/Set functions below, add a group start and group end marker above and below - like this:

https://github.com/Slicer/Slicer/blob/fab4ec043672efc3434ae29fea498979b9ae7b17/Modules/Loadable/Markups/MRML/vtkCurveGenerator.h#L54-L61